.devcontainer/post_create.sh

```diff
@@ -6,6 +6,7 @@ git config --global --add safe.directory /workspaces/dspy
 
 pip install poetry==1.7.1
 poetry config installer.max-workers 4
+poetry config virtualenvs.in-project true
```

Comment:
This just makes it so the poetry venv is created within the workspace. This, combined with the additional setting specified in `devcontainer.json`, makes it more likely that VSCode picks it up as the correct interpreter. If someone has previously selected a different interpreter for this workspace, this will not override that selection but it will make changing it to the correct interpreter a bit easier.